indico/modules/events/reminders/forms.py

```diff
@@ -56,6 +56,9 @@ class ReminderForm(IndicoForm):
     message = TextAreaField(_('Note'), description=_('A custom message to include in the email.'))
     include_summary = BooleanField(_('Include agenda'),
                                    description=_("Includes a simple text version of the event's agenda in the email."))
+    include_description = BooleanField(_('Include description'),
+                                       description=_("Includes a simple text version of the event's"
+                                                     " description in the email."))
```

Comment:
we usually keep the space in the previous line instead of starting the continued string literal with a space in the new line

Reply:
done